src/runtimes/node/index.ts

```diff
@@ -151,9 +151,9 @@ const zipFunction: ZipFunction = async function ({
     bundler: bundlerName,
     bundlerWarnings,
     config,
-    displayName: config?.name,
+    displayName: staticAnalysisName ?? config?.name,
     entryFilename: zipPath.entryFilename,
-    generator: config?.generator || getInternalValue(isInternal),
+    generator: staticAnalysisGenerator ?? config?.generator ?? getInternalValue(isInternal),
```

Comment:
By changing `||` to `??`, you're saying that an empty string might take precedence over a non-empty value. I don't think this should be the case.

```suggestion
    generator: staticAnalysisGenerator || config?.generator || getInternalValue(isInternal),
```